### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by Brenden-Trieu**
Updating readme with current information updated on Jan 13th, 2023

### CODE REVIEW

**Comment by Brenden-Trieu** [comment]
Close #21

**Review by aliL2000 — APPROVED** [review]
Nice one, looks good to me!

File: README.md [hunk 1]
```diff
@@ -1 +1,93 @@
...
+
+## Milestones
+
+1. Gathering and Confirmation of Requirements
+   Confirm the project's purpose
+   Confirm client requested functionality for project
+   Confirm tech stack
+   (i.g. Frontend, backend, Database, and software utilities - blender, algorithms)
+   Confirm integration, and potential authentication
+   Completion by the end October 15th, 2022
```
Comment (Jeena01) [inline README.md:17]: stunning very good work

File: README.md [hunk 2]
```diff
@@ -1 +1,93 @@
...
+
+2. Design
+   Get design convention
+   Share and collaborate on graphical design
+   Review and confirm app layout
+   Refine layout/interface for user functionality
+   Completion by the end October 30th, 2022
+
+3. Pre-Algorithm Coding
```
Comment (Jeena01) [inline README.md:26]: great work here

File: README.md [hunk 3]
```diff
@@ -1 +1,93 @@
...
+   Two tab based web app
+   Imports
+   Image imports will derive from local machine as database
+   Classification
+   Classifications must be user defined
+   Will be defined after importing images
+   Option to add classes will be available at anytime after initial process is completed
+   Completion by January 20th, 2023
+
+4. Algorithm Implementation (VGG16)
```
Comment (Jeena01) [inline README.md:36]: slay

**Review by Jeena01 — APPROVED** [review]
LGTM

**Comment by Jeena01** [comment]
showstopping spectacular

**Comment by Brenden-Trieu** [comment]
Documentation from old repo is still to be done

**Pull request merged**